app/assets/javascripts/discourse/app/templates/components/generated-invite-link.hbs

```diff
@@ -1,5 +1,7 @@
 <p>{{i18n "user.invited.link_generated"}}</p>
 <p>
-  <input value={{link}} class="invite-link-input" style="width: 75%" type="text">
+  <input value={{link}} class="invite-link-input" style="width: 100%" type="text">
```

Comment:
can you try to remove this inline style while you are at it please ?

Reply:
Sure, done!